docs/source/data-types/timeuuid.md

```diff
@@ -19,6 +19,41 @@ session
     .query("INSERT INTO keyspace.table (a) VALUES(?)", (to_insert,))
     .await?;
 
+// Read timeuuid from the table
+if let Some(rows) = session.query("SELECT a FROM keyspace.table", &[]).await?.rows {
+    for row in rows.into_typed::<(CqlTimeuuid,)>() {
+        let (timeuuid_value,): (CqlTimeuuid,) = row?;
+    }
+}
+# Ok(())
+# }
+```
+
+To use the Timeuuid on `uuid` crate, enable the feature `v1` in your crate using:
+
+```shell
+cargo add uuid -F v1
+```
+
+and now you're gonna be able to use the `uuid::v1` features: 
+
+```rust
+# extern crate uuid;
+# extern crate scylla;
+# use scylla::Session;
+# use std::error::Error;
+# use std::str::FromStr;
+use scylla::IntoTypedRows;
+use scylla::frame::value::CqlTimeuuid;
+use uuid::Uuid;
+# async fn check_only_compiles(session: &Session) -> Result<(), Box<dyn Error>> {
+
+// Insert some timeuuid into the table
+let to_insert = CqlTimeuuid::from(Uuid::now_v1(&[1, 2, 3, 4, 5, 6]));
```

Comment:
This can't be left here as-is; `node_id` argument for `now_v1` must be some reasonable number. We need to understand what is commonly chosen to served as `node_id` in existing applications.

Reply:
```rust
// Tip: you can use random stable numbers or your MAC Address
let node_id = [0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC];
```

Just added more context to this line just in case. Maybe we can add a new method that generates the Uuid v1 with the MAC Address or some other stable number around Hardware ID.

Maybe I'm a lazy developer, however I would love to only call a `uuid::now_v1()` without any parameter needed.

Reply:
> Maybe I'm a lazy developer, however I would love to only call a uuid::now_v1() without any parameter needed.

It's unfortunately hard to have strong guarantees without significant effort from the developer side...

Reply:
> ```rust
> // Tip: you can use random stable numbers or your MAC Address
> let node_id = [0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC];
> ```
> 
> Just added more context to this line just in case. Maybe we can add a new method that generates the Uuid v1 with the MAC Address or some other stable number around Hardware ID.

@Lorak-mmk WDYT?

Reply:
I 

> > ```rust
> > // Tip: you can use random stable numbers or your MAC Address
> > let node_id = [0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC];
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Just added more context to this line just in case. Maybe we can add a new method that generates the Uuid v1 with the MAC Address or some other stable number around Hardware ID.
> 
> @Lorak-mmk WDYT?

I think this is an example that presents how to create one's own CqlTimeuuid value from uuid, not uuid library best practices. Randomly generated node id is fine, hardcoded (like in the current version) is ok too.